docs/Paymaster/integration.mdx

```diff
@@ -55,6 +55,20 @@ const paymaster: IPaymaster = new BiconomyPaymaster({
 });
 ```
 
+## New Integration Example
+You can now assign a paymaster by providing only the paymaster api key to the smart account. By default this will use Sponsored mode.
+
```

Comment:
we don't need to update this, given this is how we want the users to create paymaster instance.